doc/commands.md

```diff
@@ -0,0 +1,41 @@
+# CLI commands documentation
+
+For each function the get_parser method is the same:
+* The function can add a node, network, or port, depending on the class. 
+* The parser is then returned. 
+
+## Node/Network
+
+Links to the commands are here:
+* [Attach](#attach)
+* [Detach](#detach)
+* [List](#list)
+
+
+### <a name="attach"></a>Attach
+
+The attach command will attack a network to a node. 
+
+* `openstack esi node network attach <node> --port <port> --network <network>` 
+    * Exactly one port or network is needed.
+    * port: name or UUID of the port.
+    * node: name or UUID of the node.
+    * network: name or UUID of the network.
+* Returns the node, port address, fixed IP, IP address and which network was attached to the node.
+
+### <a name="detach"></a>Detach
+
+The detach command will detach a network from a node. 
+
+* `openstack esi node network detach <node> <port>`
+    * port: name or UUID of the port.
+    * node: name or UUID of the node.
+
+### <a name="list"></a>List
+
+The list command will list the networks that are attached to a node. 
+
+* `openstack esi node network list --node <node> --network <network>`
+    * port: name or UUID of the port.
+    * node: name or UUID of the node.
```

Comment:
And here. Also, this command doesn't take `port`; it takes `network`. It probably makes sense to also add some text explaining that these parameters filter the output.

Reply:
I said that the given info determines the output.